tools/amp-validation/index.js

```diff
@@ -26,10 +24,12 @@ function checkEndpoints(devChannel) {
 }
 
 function runValidator(validator, devChannel) {
+    const isDev = process.argv[2] || false;
```

Comment:
not crucial but when I see `node index.js true` I have no idea what the second argument `true` means.
Can we make it more explicit? iex: `node index.js --dev`